plugins/aws_workspaces/plugin.spec.yaml

```
@@ -2,23 +2,44 @@ plugin_spec_version: v2
 extension: plugin
 products: [insightconnect]
 name: aws_workspaces
-title: AWS WorkSpaces
+title: "AWS WorkSpaces"
 vendor: rapid7
 support: community
 status: []
+sdk:
+  type: slim
+  version: 6.1.4
```

Imo we can bump it to `6.2.0`
```suggestion
  version: 6.2.0
```